### ISSUE TITLE
global is gone

### ISSUE THREAD

**Issue by donsizemore** [gdcc/dv-metrics#14]
@PaulBoon will there be a separate repo for the global metrics app (or is there), and is there documentation on installing the global app? thank you.

**Comment by PaulBoon** [comment]
@donsizemore The local part of https://github.com/IQSS/dataverse-metrics was split of into this `dv-metrics`. I assumed that in `dataverse-metrics` that local part would be removed and that it would become  a 'global only' metrics. 
I don't have any plans of doing this because my main concern is the 'local' metrics. 

@pdurbin Are there any plans for the 'global' metrics and the `dataverse-metrics` repo?

**Comment by pdurbin** [comment]
Yes. Well, not a concrete plan with a deadline or anything but I created this issue:

- https://github.com/IQSS/dataverse-metrics/issues/92

@donsizemore if you're interested in helping with that effort, let me know! 😄
